pandora-common/src/space/space.ts

```diff
@@ -61,9 +61,17 @@ export const SpaceInviteSchema = z.object({
 	characterId: CharacterIdSchema.optional(),
 	/** The time when the invite expires */
 	expires: z.number().int().optional(),
+	/** Type of the invite */
+	type: z.enum(['joinMe', 'spaceBound']),
+	/** Creator of the invite */
+	createdBy: z.object({
+		accountId: AccountIdSchema,
+		characterId: CharacterIdSchema,
+		characterName: z.string(),
```

Comment:
I'm not fully sure about storing a name of the character as well. Is it really needed? I think I would prefer only showing ID right now with ability to get it separately for display later.